src/doc/rustc/src/check-cfg.md

```diff
@@ -14,6 +14,9 @@ whether to check conditions and how to check them.
 > **Note:** No implicit expectation is added when using `--cfg`. Users are expected to
 pass all expected names and values using the _check cfg specification_.
 
+For interacing with this through Cargo,
```

Comment:
Is this interacting, or interfacing, or something else?

Reply:
interacting